llvm/docs/CommandGuide/lit.rst

```diff
@@ -399,6 +399,11 @@ ADDITIONAL OPTIONS
  Show all features used in the test suite (in ``XFAIL``, ``UNSUPPORTED`` and
  ``REQUIRES``) and exit.
 
+.. option:: --update-tests
+
+ Pass failing tests to functions in the ``lit_config.update_tests`` list to
```

Comment:
do you mean "to functions in the `list_config.test_updaters` list"? `lit_config.update_tests` is a boolean value.